phalcon/Mvc/Application.zep

```diff
@@ -79,13 +79,17 @@ class Application extends AbstractApplication
     /**
      * Handles a MVC request
      */
-    public function handle(string! uri) -> <ResponseInterface> | bool
+    public function handle(var! uri)
     {
         var container, eventsManager, router, dispatcher, response, view,
             module, moduleObject, moduleName, className, path, implicitView,
             returnedResponse, controller, possibleResponse, renderStatus,
             matchedRoute, match;
 
+        if unlikely typeof uri != "string" {
+            throw new Exception("Argument should be a string");
```

Comment:
Better use `handle(string! uri)`

Reply:
Ok will update the exception. We need to change this.
https://github.com/phalcon/cphalcon/blob/a48b31967a1ce27c10cbed61bee42e0c90addc86/phalcon/Application/AbstractApplication.zep#L97
https://github.com/phalcon/cphalcon/blob/a48b31967a1ce27c10cbed61bee42e0c90addc86/phalcon/Mvc/Application.zep#L82
https://github.com/phalcon/cphalcon/blob/a48b31967a1ce27c10cbed61bee42e0c90addc86/phalcon/Cli/Console.zep#L37

Reply:
I propose just remove `  abstract public function handle(); `. What this solves? It is not an interface, just restriction for users, isn't?